src/atlas3.js

```diff
@@ -624,6 +629,7 @@ export class Atlas3 extends MetricsPanelCtrl {
                     name: ctrl.panel.name[i],
                     max: ctrl.panel.max[i],
                     min: ctrl.panel.min[i],
+                    lineWidth: 8,
```

Comment:
LineWidth is a property of the object. It was there but it was fixed and not used in the code so the linewidth parameter didn't have anything to do with the linewidth (Because the width was hardcoded based on the zoom). I just set it up to get the value dynamically from the width multiplier that we get from the editor menu. I think this is the correct way of using this parameter. But we can talk tomorrow and I can change the name to something else if there is a reason to keep both.